index.js

```diff
@@ -1,244 +0,0 @@
-import { querySelect, querySelectAll, createElem } from "./utils.js";
-
-const cvc = querySelectAll(".security-number");
-const cardNum = querySelectAll(".card-number");
-const cardName = querySelectAll(".card-name");
-const expMonth = querySelectAll(".exp-month");
-const expYear = querySelectAll(".exp-year");
-
-const nameInput = querySelect("#user-name");
-const cardNumInput = querySelect("#user-card-number");
-const expMonthInput = querySelect("#exp-date-month");
-const expYearInput = querySelect("#exp-date-year");
-const cvcInput = querySelect("#cvc");
-
-const submitFormBtn = querySelect(".submit-form");
-const form = querySelect("form");
-const thankYouState = querySelect(".thank-you-state");
-const continueBtn = querySelect("#continue-btn");
-
-//Changes the informations on the card.
-const cardInfoChange = (input, changingElem) => {
-  changingElem.forEach((elem) => {
-    elem.innerText = input.value.toUpperCase();
-  });
-};
-
-//Limits the characters in the input.
-const charLimiter = (input, quantity) => {
-  if (input.value.length >= quantity) {
-    input.value = input.value.slice(0, quantity);
-  }
-};
-
-//Limits date inputs so the user cannot provide more than requested number in input field.
-const dateLimit = (input, quantity) => {
-  if (input.value > quantity) {
-    input.value = "";
```

Comment:
**question:** How does it work? If the user exceeds the quantity then the input clears? If so, it isn't a good solution in terms of UX

Reply:
I know but I think I didn't have an idea how to stop the user from typing something like 13th month and in the design I haven't seen the option with the select tag. Other option is to make the validation catch it and throw an error. That's my idea for it now, but probably you have a better one for this problem.